schema/03-staking.sql

```diff
@@ -82,3 +77,15 @@ CREATE TABLE validator_voting_power
     height            BIGINT NOT NULL,
     PRIMARY KEY (consensus_address, height)
 );
+
+CREATE TABLE validator_description
+(
+    operator_address      TEXT NOT NULL,
+    moniker               TEXT ,
+    identity              TEXT,
+    website               TEXT,
+    security_contact      TEXT,
+    details               TEXT,
+    height                BIGINT,
+    timestamp             TIMESTAMP
```

Comment:
What about switching from `TIMESTAMP` to `TIMESTAMP WITHOUT TIME ZONE`? I would also add the `NOT NULL` clause here. 

```suggestion
    timestamp             TIMESTAMP WITHOUT TIME ZONE NOT NULL
```